ECIPs/ECIP-1057.md

```diff
@@ -0,0 +1,106 @@
+### Title
+
+```
+    ECIP: 1057
+    Title: Cold Staking
+    Author: Dexaran, dexaran@ethereumclassic.org
+    Discussions-To: dexaran@ethereumclassic.org
```

Comment:
Can you change `discussions-to` to point to issue #65

Reply:
Sure